src/output_writers.jl

```diff
@@ -324,3 +324,72 @@ function time_to_write(clock, out::JLD2OutputWriter)
         return false
     end
 end
+
+mutable struct Checkpointer <: OutputWriter
+                 dir :: String
+              prefix :: String
+    output_frequency :: Int
+end
+
+function Checkpointer(; output_frequency, dir=".", prefix="checkpoint", force=false)
```

Comment:
The kwarg 'force' is not used here. It should either be used, or removed from the function signature.

In the `JLD2OutputWriter`, the kwarg `force` indicates whether file creation should be 'forced' (it corresponds to the same keyword passed to `mkpath`.